src/olympia/versions/models.py

```diff
@@ -185,7 +184,8 @@ def should_have_due_date(self, negate=False):
                 addon__promotedaddon__group_id__in=(
                     g.id for g in PROMOTED_GROUPS if g.listed_pre_review
                 )
-            ),
+            )
+            | Q(addon__type__in=amo.GROUP_TYPE_THEME),
             addon__status__in=(amo.VALID_ADDON_STATUSES),
```

Comment:
`VALID_ADDON_STATUS` contains only `(STATUS_NOMINATED, STATUS_APPROVED)` so that's what makes this work.

Reply:
What does "this" refer to?